tools/postgres-init-dev-db

```diff
@@ -24,7 +24,7 @@ if [[ $# == 0 ]]; then
     USERNAME=zulip
     PASSWORD=$("$(dirname "$0")/../bin/get-django-setting" LOCAL_DATABASE_PASSWORD)
     DBNAME=zulip
-    SEARCH_PATH="$USERNAME",public
+    SEARCH_PATH="$USERNAME",public,pgroonga,pg_catalog
```

Comment:
From our discussion on provision.py, it sounds like you're planning to defer enabling this feature in production for a follow-up PR; since this is related to the same goal, it's fine for this to be bundled with that follow-up issue.